src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { createTodo } from '../../api/todos';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  setTodos: React.Dispatch<React.SetStateAction<Todo[]>>;
+  setErrorMessage: (v: string) => void;
+};
+
+export const NewTodo: React.FC<Props> = React.memo(({
+  setTodos, setErrorMessage,
+}) => {
+  const [query, setQuery] = useState('');
+
+  const addTodo = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (query.trim() === '') {
+      setQuery('');
+
+      return;
+    }
+
+    createTodo({
+      userId: 11253,
```

Comment:
Hardcoding 'userId' is not a good practice. It should be dynamic or taken from the user's session or state.